docs.html

```diff
@@ -0,0 +1,552 @@
+                <li class="navbar-links-list-item"><a href="/docs.html">Docs</a></li>
+                <li class="navbar-links-list-item"><a href="https://github.com/shanudjn/builder-ui">Github</a></li>
+            </ul>
+        </div>
+    </nav>
+    <aside class="aside">
+
+        <ul>
+            <li class="aside-list-item"><a href="#installation"><b>Installation</b></a></li>
+
+            <li class="aside-list-item"><a href="#alert">Alert</a></li>
+            <li class="aside-list-item"><a href="#avatar">Avatar</a></li>
+            <li class="aside-list-item"><a href="#badge">Badge</a></li>
+            <li class="aside-list-item"><a href="#cards">Cards</a></li>
+            <li class="aside-list-item"><a href="#grids">Grids</a></li>
+            <li class="aside-list-item"><a href="#input">Input</a></li>
+            <li class="aside-list-item"><a href="#list">Lists</a></li>
+            <li class="aside-list-item"><a href="#navigation">Navigation</a></li>
+            <li class="aside-list-item"><a href="#ratings">Ratings</a></li>
+            <li class="aside-list-item"><a href="#text">Text</a></li>
+        </ul>
+    </aside>
+    <div class="doc">
+        <!--Installation-->
+        <h2 id="installation">Getting Started</h2>
+        <div class="code-snippet">
+            <pre>
+                <code>
+//link to the stylesheet by adding the following line
+
+&ltlink rel="stylesheet"
+ href="https://builder-ui-css.netlify.app/styles/components/styles.css"&gt
+                </code>
+            </pre>
+        </div>
+        <!--Alert-->
+        <h2>Alert</h2>
+        <p class="paragraph-text">Documentation and Examples on BuilderUI Alerts</p>
+        <h4>Examples</h4>
+        <div class="div-example">
+            <div>
+                <p class="alert alert-error">
+                    There was an error processing your request
+                </p>
+                <p class="alert alert-success">
+                    Data uploaded to server. Fire on!
+                </p>
+                <p class="alert alert-warning">
+
+                    There's something wrong. Could you please check again.
+                </p>
+                <p class="alert alert-info">
+
+                    There's something wrong. Could you please check again.
+                </p>
+            </div>
+        </div>
+        <h4>Code Examples</h4>
+        <div class="code-snippet">
+            <pre><code class="language-html">
+        &ltp class="alert alert-error"&gt
+            There was an error processing your request
+        &lt/p&gt
+        &ltp class="alert alert-success"&gt
+            Data uploaded to server. Fire on!
+        &lt/p&gt
+        &ltp class="alert alert-warning"&gt
+            There's something wrong. Could you please check again.
+        &lt/p&gt
+        &ltp class="alert alert-info"&gt
+            There's something wrong. Could you please check again.
+        &lt/p&gt
```

Comment:
Code snippets can be made more appealing using libraries like [Prism js](https://prismjs.com/) or, [Highlights js](https://highlightjs.org/)